src/Angor/Avalonia/AngorApp/Sections/Wallet/Create/Step_2/SeedWordsViewModel.cs

```diff
@@ -0,0 +1,36 @@
+using System.Linq;
+using System.Reactive.Linq;
+using AngorApp.Model;
+using AngorApp.Sections.Browse;
+using CSharpFunctionalExtensions;
+using ReactiveUI.SourceGenerators;
+using ReactiveUI.Validation.Helpers;
+
+namespace AngorApp.Sections.Wallet.Create.Step_2;
```

Comment:
As suggested we never really `_` for types or ns only maybe for params, rename this to Step2